clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -83,14 +83,14 @@ static Attr *handleIntelFPGALoopAttr(Sema &S, Stmt *St, const ParsedAttr &A) {
 
   unsigned NumArgs = A.getNumArgs();
   if (NumArgs > 1) {
-    S.Diag(A.getLoc(), diag::err_attribute_too_many_arguments) << A << 1;
```

Comment:
Just to confirm: `err_attribute_too_many_arguments` is still used somewhere else in the code base. Right?

Reply:
Yes. This is general clang error message, that wasn't be added as a part of FPGA loop attributes activities, used 4 times in SemaDeclAttr.